app/controllers/concepts.ts

```diff
@@ -18,12 +19,24 @@ export default class ConceptsController extends Controller {
   @service declare router: RouterService;
   @service declare authenticator: AuthenticatorService;
 
+  get currentUser(): UserModel | null {
```

Comment:
If the return type says `UserModel | null`, why are we returning with `as UserModel`?